src/encoded/static/components/browse/components/SearchResultTable.js

```diff
@@ -118,18 +116,16 @@ class ResultDetail extends React.PureComponent{
     render(){
         var { open, rowNumber, result, tableContainerWidth, tableContainerScrollLeft, renderDetailPane, toggleDetailOpen } = this.props;
         return (
-            <div className={"result-table-detail-container" + (open || this.state.closing ? ' open' : ' closed')}>
+            <div className={"result-table-detail-container detail-" + (open || this.state.closing ? 'open' : 'closed')}>
```

Comment:
Renaming "open" to "detail-open" className here and in SCSS because `.open button.dropdown-button` is a CSS selector for a Bootstrap style for when dropdown is open which gets erroneously activated otherwise.